<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDialog</class>
 <widget class="QDialog" name="TableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/res/64.ico</normaloff>:/images/res/64.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background: white;
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>CN (Common Name):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Serial Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_cer_validafter">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Expiry Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_subject_o">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_cer_validbefore">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>C (Country):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_name_cn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>OU (Organization Unit):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>L (Locality):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_cer_version">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>C (Country):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_subject_c">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_name_c">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: #007A6F;</string>
   </property>
   <property name="text">
    <string>Issuer Certificate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: #007A6F;</string>
   </property>
   <property name="text">
    <string>Issuer Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_subject_l">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_name_ou">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_issue_name_o">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_subject_cn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>O (Organization):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>O (Organization):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Effective Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>CN (Common Name):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>background: transparent;
color: #007A6F;</string>
   </property>
   <property name="text">
    <string>Subject Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Key Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pubkey_alg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Public Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pubkey_keysz">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: #007A6F;</string>
   </property>
   <property name="text">
    <string>Public Key Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Key Algorithm:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editT_pubkey_value">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #007A6F;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editT_pubkey_serial">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #007A6F;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
